Cargo.lock

```diff
@@ -2642,6 +2642,17 @@ dependencies = [
  "digest 0.9.0",
 ]
 
+[[package]]
+name = "hmac-drbg"
```

Comment:
Should it be removed?

Reply:
Why? There are packages which depend on it.